The latest changes added this clang-tidy warning. I am fine with fixing it.
```warning: macro replacement list should be enclosed in parentheses [bugprone-macro-parentheses]
scwx-qt/source/scwx/qt/gl/draw/geo_lines.hpp:22:4: warning: use 'using' instead of 'typedef' [modernize-use-using]
   22 |    typedef std::function<void(const std::shared_ptr<GeoLineDrawItem>&,
      |    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |    using HoverCallback = std::function<void (const std::shared_ptr<GeoLineDrawItem> &, const QPointF &)>
   23 |                               const QPointF&)>
      |                               ~~~~~~~~~~~~~~~~
   24 |       HoverCallback;
      |       ~~~~~~~~~~~~~```

--- a/scwx-qt/source/scwx/qt/gl/draw/geo_lines.hpp
+++ b/scwx-qt/source/scwx/qt/gl/draw/geo_lines.hpp
@@ -19,9 +19,8 @@ struct GeoLineDrawItem;
 class GeoLines : public DrawItem
 {
 public:
-   typedef std::function<void(const std::shared_ptr<GeoLineDrawItem>&,
-                              const QPointF&)>
-      HoverCallback;
+   using HoverCallback = std::function<void(
+      const std::shared_ptr<GeoLineDrawItem>&, const QPointF&)>;
 
    explicit GeoLines(std::shared_ptr<GlContext> context);
    ~GeoLines();

